internal/comments/generate.go

```diff
@@ -390,6 +375,49 @@ func generateStructFromParams(origin string, s *Statement, name string, params [
 	return nil
 }
 
+func handleCommonObjects(origin, fieldName string) (string, *keyInfo) {
+	if !strings.Contains(fieldName, ".") {
+		return "", nil
+	}
+
+	// key prefix to manually declared struct in typedefs package
+	lookup := map[string][]string{
+		"bounds.":          {"Bounds", "The bounding box of the object (source, scene item, etc)."},
+		"crop.":            {"Crop", "The crop specification for the object (source, scene item, etc)."},
+		"position.":        {"Position", "The position of the object (source, scene item, etc)."},
+		"scale.":           {"Scale", "The scaling specification for the object (source, scene item, etc)."},
+		"item.":            {"Item", "The item specification for this object."},
+		"font.":            {"Font", "The font specification for this object."},
+		"settings.":        {"Settings", " "},
+		"stream.settings.": {"Settings", " "},
+		"items.*.":         {"Item", "The items for this object."},
+		"filters.*.":       {"Filter", "The filters for this object."},
+	}
+
+	for k, vs := range lookup {
+		if strings.HasPrefix(fieldName, k) {
+			k = strings.TrimSuffix(k, ".")
+			s := Qual(typedefQualifier(origin), vs[0])
+			return k, &keyInfo{Type: s, Comment: vs[1]}
+		}
+	}
+
+	fmt.Printf("  > not handled as a common struct: %s\n", fieldName)
+
+	return "", nil
+}
```

Comment:
For posterity:

This selection is pretty much the meat of the PR. Here is what's doing the matching and selecting our manually written structs.